### ISSUE TITLE
PDP Prod Error:  My Team page error when direct report/shared employee have no calculated next conversation due date entry

### ISSUE THREAD

**Issue by telusdcinco** [bcgov/performance#924]
PDP Prod Error:  My Team page error when direct report/shared employee have no calculated next conversation due date entry

![image.png](https://images.zenhubusercontent.com/60e88ef619e4760c8b3ee8bb/1148c8dd-22e4-401c-8e87-b3f11d373497)

**Comment by telusdcinco** [comment]
@jizhaogit  I've made the fix.  Please review.

**Comment by telusdcinco** [comment]
To test this issue, add employee 055479 to manager 158932.

**Comment by telusdcinco** [comment]
@jizhaogit Just realized that your last code fix is not in prod.  We'll just need to push line 61 to prod.

![image.png](https://images.zenhubusercontent.com/60e88ef619e4760c8b3ee8bb/c5faee4d-9dbf-47bb-ada3-e5d41ef6a74f)

**Comment by jizhaogit** [comment]
@telusdcinco yes i recalled the fix. but not sure which ticket it belonged with...   #817 has no code changes because it related with the employee and employee_jr table changes.

**Comment by telusdcinco** [comment]
@jizhaogit There were code changes for #817 migrated on the week of Oct 25, 2022.  The change for the employee demo jr fixed the issue.  Apparently, 1 line change did not make it to prod.
I just added a space to the line so it gets recognized with change, and gets pushed to prod.

**Comment by jizhaogit** [comment]
@telusdcinco thank you for the verification!

**Comment by steffipravasi** [comment]
The issue is resolved in the test as it was a missing line of code in the prod environment. Dennis and I validated the ticket together on test and Michael Rosero does not receive the Ajax error when Team Members in MyTeams is opened. 
Moving this ticket to Travis to verify it in Prod environmet.
